src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -607,13 +611,20 @@ function prepareCluster(cluster, context, isExtension = false) {
         isOptional: attribute.$.optional == 'true',
         reportingPolicy: reportingPolicy,
         storagePolicy: storagePolicy,
-        isSceneRequired: attribute.$.sceneRequired == 'true',
+        isSceneRequired:
+          attribute.$.sceneRequired == 'true' ||
+          (quality != null && quality.scene == 'true'),
         introducedIn: attribute.$.introducedIn,
         removedIn: attribute.$.removedIn,
         isNullable: attribute.$.isNullable == 'true' ? true : false,
         entryType: attribute.$.entryType,
         mustUseTimedWrite: attribute.$.mustUseTimedWrite == 'true',
         apiMaturity: attribute.$.apiMaturity,
+        changeOmitted: quality ? quality.changeOmitted == 'true' : false,
+        //nullable:  quality ? quality.nullable == 'true' : false, -> left for new Data model xml
+        //scene:  quality ? quality.scene == 'true' : false, -> left for new Data model xml
+        persistence: quality ? quality.persistence : null,
+        //reportable:  quality ? quality.reportable == 'true' : false, -> left for new Data model xml
```

Comment:
Generally we call booleans `isSomething`. So we should call this `isReportable`?

Reply:
May be I should get rid of the ones I have commented here. Does not make sense.